packages/@react-spectrum/menu/stories/MenuTrigger.stories.tsx

```diff
@@ -41,67 +87,310 @@ let withSection = [
 ];
 
 storiesOf('MenuTrigger', module)
+  .addParameters({chromaticProvider: {colorSchemes: ['light']}, chromatic: {pauseAnimationAtEnd: true}})
+  .add(
+    'default menu (static)',
+    () => render(
+      <Menu onAction={action('onAction')}>
+        <Item>One</Item>
+        <Item>Two</Item>
+        <Item>Three</Item>
+      </Menu>
+    )
+  )
+  .add(
+    'default menu (generative)',
+    () => render(
+      <Menu items={flatMenu} onAction={action('onAction')}>
+        {item => <Item key={item.name}>{item.name}</Item>}
+      </Menu>
+    )
+  )
+  .add(
+    'default menu w/ section (static)',
+    () => render(
+      <Menu onAction={action('onAction')}>
+        <Section title="Section 1">
+          <Item>One</Item>
+          <Item>Two</Item>
+          <Item>Three</Item>
+        </Section>
+        <Section title="Section 2">
+          <Item>One</Item>
+          <Item>Two</Item>
+          <Item>Three</Item>
+        </Section>
+      </Menu>
+    )
+  )
+  .add(
+    'default menu w/ section (generative)',
+    () => render(defaultMenu)
+  )
+  .add(
+    'default menu w/ titleless sections (static)',
+    () => render(
+      <Menu onAction={action('onAction')}>
+        <Section aria-label="Section 1">
+          <Item>One</Item>
+          <Item>Two</Item>
+          <Item>Three</Item>
+        </Section>
+        <Section aria-label="Section 2">
+          <Item>One</Item>
+          <Item>Two</Item>
+          <Item>Three</Item>
+        </Section>
+      </Menu>
+    )
+  )
+  .add(
+    'default menu w/ titleless sections (generative)',
+    () => render(
+      <Menu items={withSection} onAction={action('onAction')}>
+        {item => (
+          <Section key={item.name} items={item.children} aria-label={item.name}>
+            {item => <Item key={item.name}>{item.name}</Item>}
+          </Section>
+        )}
+      </Menu>
+    )
+  )
+  .add(
+    'Static with dialog trigger',
+    () => render(
+      <Menu>
+        <Section title="Actions">
+          <DialogTrigger isDismissable>
+            <Item>Edit...</Item>
+            <Dialog>
+              <Header>
+                <Heading>Edit</Heading>
+              </Header>
+              <Divider size="M" />
+              <Content>
+                Testing
+              </Content>
+            </Dialog>
+          </DialogTrigger>
+          <DialogTrigger>
+            <Item>Delete...</Item>
+            <AlertDialog title="Delete" variant="destructive" primaryActionLabel="Delete" cancelLabel="Cancel">
+              Are you sure?
+            </AlertDialog>
+          </DialogTrigger>
+        </Section>
+      </Menu>
+    , {isOpen: true}),
```

Comment:
should isOpen be specified in the name? i just came over here to make sure that's how it was supposed to be

Reply:
So that kinda depends (I think it might be a bug, but unsure how we expect users to do something like this). Without the `isOpen`, the Dialog actually will render and then disappear (I think because the Menu closes -> DialogTrigger button is no longer rendered -> modal unmounts). Putting `closeOnSelect='false'` doesn't work either, the menu gets closed when the Dialog renders.

Reply:
@devongovett @dannify any thoughts on the above?